frontend/src/components/common/EditModuleModal.tsx

```diff
@@ -72,13 +86,15 @@ const EditModuleModal = ({
             label="Module Name:"
             defaultValue={title}
             onChange={setTitle}
+            isInvalid={invalid}
             isRequired
+            helperText={`${title.length}/${MAX_TITLE_CHARACTERS} characters`}
+            errorMessage={`${title.length}/${MAX_TITLE_CHARACTERS} characters`}
```

Comment:
Is this a good UX if the title is empty? Might be clearer to say "This field is required" in that case.